waggle-dance-core/src/main/java/com/hotels/bdp/waggledance/client/compatibility/HiveThriftMetaStoreIfaceCompatibility1xx.java

```diff
@@ -29,72 +26,38 @@
 import org.apache.hadoop.hive.metastore.api.NoSuchObjectException;
 import org.apache.hadoop.hive.metastore.api.PrimaryKeysRequest;
 import org.apache.hadoop.hive.metastore.api.PrimaryKeysResponse;
-import org.apache.hadoop.hive.metastore.api.Table;
-import org.apache.hadoop.hive.metastore.api.ThriftHiveMetastore;
 import org.apache.hadoop.hive.metastore.api.UnknownDBException;
 import org.apache.thrift.TException;
 
-import lombok.AllArgsConstructor;
-
-@AllArgsConstructor
-public class HiveThriftMetaStoreIfaceCompatibility1xx implements HiveThriftMetaStoreIfaceCompatibility {
-
-  private final ThriftHiveMetastore.Client client;
+/**
+ * This interface contains methods that are missing from Hive 1.x.x but added in later version of Hive
+ */
+public interface HiveThriftMetaStoreIfaceCompatibility1xx {
```

Comment:
Nit: should this be simplified to just 1x? The patch portion won't really make a difference.